index.md

```diff
@@ -1,113 +1,253 @@
 
 <div class="intro-text">
-  The <a href="https://github.com/ethereumjs/">EthereumJS</a> community builds Javascript tools implementing core <a href="https://ethereum.org">Ethereum</a>
-  technologies, protocols and APIs for helping developers to interact with the Ethereum network and build
-  their own applications.
+  <a href="https://github.com/ethereumjs/">EthereumJS</a> is an implementation of the core <a href="https://ethereum.org">Ethereum</a> Execution Layer (EL) Protocol stack in TypeScript.
 </div>
 
+<div class="intro-text">
+  <a href="https://github.com/ethereumjs/ethereumjs-monorepo">https://github.com/ethereumjs/ethereumjs-monorepo</a> 
+  &nbsp;{% include repository_small.html name="ethereumjs-monorepo" %}
+</div> 
+
 <h1>Projects</h1>
 
 <div class="repo-group">
-  <h3><i class="fa fa-cogs"></i> VIRTUAL MACHINE</h3>
-  <p>Implementation of the Ethereum virtual machine supporting the latest fork rules.</p>
-  {% include repository.html name="ethereumjs-vm" %}
+  <h3><i class="fa fa-cogs"></i> VIRTUAL MACHINE (EVM)</h3>
+  <p>Implementation of the Ethereum Virtual Machine in TypeScript supporting
+  all hardforks and allowing for easy integration of the EVM in browser applications.</p>
```

Comment:
Could we say "web applications" here instead of `browser`?  I know it's nitpicky but "web" sounds more hip and modern in a marketing sorta way.

Reply:
Updated (will push later).